client/src/sass/_about.scss

```diff
@@ -0,0 +1,179 @@
+/*---------------------
+  About
+-----------------------*/
+.about {
+    padding-bottom: 70px;
+}
+
+.about__pic {
+    margin-bottom: 35px;
+
+    img {
+        min-width: 100%;
+    }
+}
+
+.about__item {
+    margin-bottom: 30px;
+
+    h4 {
+        color: $heading-color;
+        font-weight: 700;
+        margin-bottom: 10px;
+    }
+
+    p {
+        margin-bottom: 0;
+    }
+}
+
+/*---------------------
+  Testimonial
+-----------------------*/
+.testimonial {
+    background: #f3f2ee;
```

Comment:
Creating variables for all colors and using them.

Replace for the whole project.